layers/+tools/shell/funcs.el

```diff
@@ -56,28 +56,27 @@
                                        100)
                                     (window-width)))))
 
-(defmacro make-shell-pop-command (func &optional shell)
+(defmacro make-shell-pop-command (name func &optional shell)
   "Create a function to open a shell via the function FUNC.
 SHELL is the SHELL function to use (i.e. when FUNC represents a terminal)."
-  (let* ((name (symbol-name func)))
-    `(defun ,(intern (concat "spacemacs/shell-pop-" name)) (index)
-       ,(format (concat "Toggle a popup window with `%S'.\n"
-                        "Multiple shells can be opened with a numerical prefix "
-                        "argument. Using the universal prefix argument will "
-                        "open the shell in the current buffer instead of a "
-                        "popup buffer.") func)
-       (interactive "P")
-       (require 'shell-pop)
-       (if (equal '(4) index)
-           ;; no popup
-           (,func ,shell)
-         (shell-pop--set-shell-type
-          'shell-pop-shell-type
-          (backquote (,name
-                      ,(concat "*" name "*")
-                      (lambda nil (,func ,shell)))))
-         (shell-pop index)
-         (spacemacs/resize-shell-to-desired-width)))))
+  `(defun ,(intern (concat "spacemacs/shell-pop-" name)) (index)
+     ,(format (concat "Toggle a popup window with `%S'.\n"
+                      "Multiple shells can be opened with a numerical prefix "
+                      "argument. Using the universal prefix argument will "
+                      "open the shell in the current buffer instead of a "
+                      "popup buffer.") func)
```

Comment:
This was not introduced by this PR, but we might as well fix it at the same time.

Move `func` to the line below to remove the `font-lock-warning-face` (orange color):
With the cursor on `func` and calling `describe-char` (`SPC h d c`), the help buffer says:
>There are text properties here:
  face                 font-lock-warning-face
  fontified            nil
  help-echo            "Hidden behind deeper element; move to another line?"

```suggestion
                      "popup buffer.")
              func)
```

Reply:
Done 👍 ✨